data_pipeline/testing_helpers/containers.py

```diff
@@ -36,7 +36,12 @@
 
 class ContainerUnavailableError(Exception):
     def __init__(self, project='unknown', service='unknown'):
-        Exception.__init__(self, "Container for project {0} and service {1} failed to start".format(project, service))
+        msg = open("logs/docker-compose.log", 'r').read()
```

Comment:
Can we have some error handling on this log opening? If no logs are found then it should print that result instead of getting an error finding this log.